### ISSUE TITLE
Cannot save config (folder protected from writing ?)

### ISSUE THREAD

**Issue by bilrousseau** [nvaccess/nvda#11139]
Hello,

I can't save my configuration in NVDA as soon as I edit the scratchpad folder.
What I've done :

- Opened C:\Users\billr\AppData\Roaming\nvda\scratchpad with VSCode
- Wrote a simple global plugin to catch NVDA+A input : (example.py)

```
# Add-on development first example

import globalPluginHandler
import tones # We want to hear beeps

class GlobalPlugin(globalPluginHandler.GlobalPlugin):

	def script_doBeep(self, gesture):
		tones.beep(440, 1000)  # Beep a standard middle A for 1 second.

	__gestures={
		"kb:NVDA+A": "doBeep"
	}
```

My plugin didn't work so I tried to save configuration in NVDA.
- NVDA+N -> Save configuration

I got the following error message : 
Cannot save configuration : file system may not authorize writing.

I tried to authorize writing in C:\Users\billr\AppData\Roaming\nvda by unchecking the readonly checkbox with "Right-click -> Properties", but the system doesn't take it into account, it stays readonly.

Problem still prevails after uninstalling / reinstalling NVDA and/or rebooting computer.

What am I missing ?

**Comment by DrSooom** [comment]
@bilrousseau: Please completely fill out the bug report draft. Thanks.

**Comment by Adriani90** [comment]
@bilrousseau are you logged in on your computer as administrator? Then you should be able to save the settings. Otherwise try to run NVDA as administrator from your local user account and see if it's working.
Please report your results.

For what I can say this seems to be a issue in your system privileges and the folder might be protected by a security software or any other procetion settings in your Windows. This is most probably not an NVDA issue.

**Comment by Adriani90** [comment]
We didn't get any updates upon request from the initial author since many years, I am closing as abandoned. if you are still having this issue, please coment and we cann reopen.
